### ISSUE TITLE
Top level context menu

### ISSUE THREAD

**Issue by ghbaud** [Pitmairen/selection-search#31]
It would be nice to be able to designate a search engine to appear in the top level of the context menu, rather than in a submenu. Is that possible?

**Comment by Pitmairen** [comment]
As far as I know this is not possible, because chrome does not allow extensions to add such items to the menu.

**Comment by ghbaud** [comment]
Take a look at ​Simple = Select + Search
<https://chrome.google.com/webstore/detail/simple-select-%20-search/aagminaekdpcfimcbhknlgjmpnnnmooo>.
If you delete all the search engines except one, that one gets promoted to
the top level. So it must be possible somehow.

**Comment by Pitmairen** [comment]
Yes it is possible if you have only a single item. I see that currently a sub menu is created even if there is only a single search engine in the context menu, so I guess I can try to fix this so that the search engine is added directly to the context menu if there is only one, but you will be limited to a single search engine in the context menu.

**Comment by ghbaud** [comment]
Actually, I was hoping for an option to have more than one in the top
level, but if it's not possible ...

Thanks for your help.

**Comment by Pitmairen** [comment]
An extension can only have a single top level item. That was the limitation before, I don't think it have changed.

**Comment by ghbaud** [comment]
So I guess the way a user would work around this issue it to have a
separate context search extension for each top-level search he needs.

Thanks again.
